financial-templates-lib/logger/ConsoleTransport.js

```diff
@@ -25,10 +23,10 @@ const customFormat = combine(
   })
 );
 
-function createConsoleTransport() {
+function createConsoleTransport(shouldColorize = true) {
   return new winston.transports.Console({
     handleExceptions: true,
-    format: customFormat
+    format: shouldColorize ? combine(customFormat, colorize()) : customFormat
```

Comment:
colorized slack console logs are great except if they are piped out from a subprocess. In this context, they add noise to the logs and so this should be optional. by default is enabled.

Reply:
Does GCP pipe them out from the subprocess?

Reply:
The `CloudResponse.js` script pipes them from the `exec` `child_process` into the HTTPS response sent back to the hub and in the Winston log message.